src/components/filter/MessagesFilterPanel.tsx

```diff
@@ -163,24 +163,24 @@ const MessagesFilterPanel = () => {
 			currentValue: currentStream,
 			setCurrentValue: setCurrentStream,
 			autocompleteList: messagesStore.messageSessions,
+			className: 'messages-window-header__session-filter',
 		};
 	}, [streams, setStreams, currentStream, setCurrentStream, messagesStore.messageSessions]);
 
-	const filterConfig: Array<FilterRowConfig> = React.useMemo(() => {
-		return [sessionFilterConfig, ...compoundFilterRow];
-	}, [sessionFilterConfig, compoundFilterRow]);
-
 	return (
-		<FilterPanel
-			isLoading={isLoading}
-			isFilterApplied={isApplied}
-			count={isApplied ? messagesDataStore.messages.length : null}
-			setShowFilter={setShowFilter}
-			showFilter={showFilter}
-			config={filterConfig}
-			onSubmit={submitChanges}
-			onClearAll={messagesStore.clearFilters}
-		/>
+		<>
+			<FilterPanel
```

Comment:
If for some reason filters are failed to fetch panel looks like this
![Screenshot from 2021-03-05 14-19-10](https://user-images.githubusercontent.com/45719774/110109032-fb19e000-7dbd-11eb-8937-2fe9ca9fda6f.png)
maybe we should handle it somehow